pyproject.toml

```diff
@@ -1,3 +1,16 @@
+[tool.setuptools]
+packages = ["x2gbfs"]
+
+[project]
+name = "x2gbfs"
+version = "0.0.1"
+dependencies = [
+    "requests",
```

Comment:
Usually, it's a good idea to pin the versions here like this: https://github.com/ParkenDD/parkapi-sources-v3/blob/main/pyproject.toml#L37